### PR TITLE
London | emmanuel gessessew| forms| WEEK 2

### PR DESCRIPTION
**Pull request by Emmanuelgessessew**
## Learners, PR Template

Self checklist

- [ yes] I have committed my files one by one, on purpose, and for a reason
- [ yes] I have titled my PR with COHORT_NAME | and | REPO | WEEK 2
- [ yes] I have tested my changes
- [ yes] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ yes] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

I applied semantic HTML to design the web page, and used CSS to style so that the form page appear decent.
## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
👋 Hi @Emmanuelgessessew  . Please complete the PR template provided and update your title to the format asked for. Look at https://github.com/CodeYourFuture/Module-User-Focused-Data/pulls to see what everyone else is doing

**Comment by Emmanuelgessessew** [comment]
I have done it. Thanks for the feedback.

File: Form-Controls/index.html [hunk 1]
```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label> Name
```
Comment (jamesbaskerville) [inline Form-Controls/index.html:18]: Typically, labels should be attached to an input explicitly -- how might you do this?

File: Form-Controls/index.html [hunk 2]
```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
...
+      <input required type="text" placeholder="Bereket Hayelom">
```
Comment (jamesbaskerville) [inline Form-Controls/index.html:19]: What validations are necessary on the name?

File: Form-Controls/index.html [hunk 3]
```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
...
+      </label>
+  </div>
+  <br>
+  <div> 
+      <label> Email
+      <input required type="email" placeholder="bereket@example.com">
+      </label>
+  </div>
+  <br>
+  <div>
```
Comment (jamesbaskerville) [inline Form-Controls/index.html:29]: How can you ensure the user picks both a color and size before ordering?

File: Form-Controls/index.html [hunk 4]
```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
...
+      <input required type="email" placeholder="bereket@example.com">
+      </label>
+  </div>
+  <br>
+  <div> 
+    <label> color
+    <select name="tshirt-color" id="tshirt-color">
+      <option value="">black</option>
+      <option value="">green</option>
+      <option value="">blue</option>
```
Comment (jamesbaskerville) [inline Form-Controls/index.html:34]: What does the `value` prop do for select inputs? Same question for the size input

File: Form-Controls/styles.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+input[type="text"],
+input[type="email"],
+input[type="date"],
+select {
+    width: 100%;
+    padding: 8px;
+    margin-top: 5px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+}
```
Comment (jamesbaskerville) [inline Form-Controls/styles.css:47]: I notice that the text and date inputs are wider than the selects -- how can we fix this?

**Review by jamesbaskerville — CHANGES_REQUESTED** [review]
Looking good so far! A few comments on requirements and visual consistency

**Comment by jamesbaskerville** [comment]
> I have done it. Thanks for the feedback.

@Emmanuelgessessew please also update your PR title to the correct format `COHORT_NAME | FULL_NAME | REPO | WEEK 2`

File: Form-Controls/index.html (hunk 1 above)
Comment (Emmanuelgessessew) [reply Form-Controls/index.html:18]: <div> 
<label for = "name" > name: < /label>
<input type="text" id="name">
<div>

File: Form-Controls/index.html (hunk 3 above)
Comment (Emmanuelgessessew) [reply Form-Controls/index.html:29]: i add the required on my html to make sure they submitted both

File: Form-Controls/styles.css (hunk 5 above)
Comment (Emmanuelgessessew) [reply Form-Controls/styles.css:47]: i can adjust the width to 50px
select {
      width: 50%;
}

**Pull request closed without merging**